grails-app/services/org/pih/warehouse/product/ProductTypeService.groovy

```diff
@@ -17,4 +19,18 @@ class ProductTypeService {
 
         return productType.sequenceNumber
     }
+
+    def getDefaultProductType() {
```

Comment:
This is fine as-is, but this could also be implemented as a named query on the ProductType domain.

http://docs.grails.org/latest/ref/Domain%20Classes/namedQueries.html

Reply:
I initially added it in the domain, but when I was calling it in the gsp template I was getting weird exception, hence I moved it here. I'll try again with domain version.

Reply:
But I have not tried a named query then, perhaps it will be better. Thanks for the suggestion.